homepage.php

```diff
@@ -52,57 +52,57 @@
 				<a href = "Query8.php"><img src = "images/Indie.jpg" class="image1"></a>
 				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
 				<div class="overlay">
-    				<div class="text"><a href = "Query8.php">Indie</a></div>
+    				<div class="text"><a href = "Query8_admin.php">Indie</a></div>
```

Comment:
2/9/2021: So I make a page called Query8_admin will mostly the same with Query8 but the only difference was in Query8_admin I use the admin nav which can access to the setting. So for admin page I should make it to link to Query8_admin.